docs/examples/ros2_compatibility.rst

```diff
@@ -0,0 +1,64 @@
+.. _examples_ros2_compatibility:
+
+ROS 2 Compatibility
+===================
+
+As stated in the :ref:`linux_ros2_compilation` section, eProsima Shapes Demo can be built and used on a ROS 2 installation with additional ROS 2 features.
+
+With ROS 2 features enabled, an additional "Use ROS2 Topics" checkbox in the Participant configuration dialog will be shown.
+
+.. image:: /01-figures/participant_ros_enabled.png
+   :scale: 100 %
+   :alt: ROS 2 Participant Configuration Options
+   :align: center
+
+Enabling this box will put Shapes Demo in ROS 2 mode.
+ROS 2 Shapes Demo main window changes to reflect this by showing a different title and the `Vulcanexus <https://www.vulcanexus.org/>`_ logo.
+
+.. image:: /01-figures/ros2_shapes_demo.png
+   :scale: 100 %
+   :alt: ROS 2 Shapes Demo Main Window
+   :align: center
+
+When using eProsima Shapes Demo in this mode, ROS 2 will be aware of the Topics transmitted.
+For instance, this is the output of the ``ros2 topic list`` command.
+
+   .. code-block:: bash
+
+      $ ros2 topic list -t
+      /Square [shapes_demo_typesupport/idl/KeylessShapeType]
+      /parameter_events [rcl_interfaces/msg/ParameterEvent]
+      /rosout [rcl_interfaces/msg/Log]
+
+Since there's a TypeSupport available for these messages, it can be used by ROS 2 to interact with the different ShapesDemo topics.
```

Comment:
```suggestion
Since there is a TypeSupport available for these messages, it can be used by ROS 2 to interact with the different ShapesDemo topics.
```